internal/concierge/server/server.go

```diff
@@ -270,5 +270,18 @@ func getAggregatedAPIServerScheme(apiGroup string) *runtime.Scheme {
 	utilruntime.Must(loginv1alpha1.RegisterConversions(scheme))
 	utilruntime.Must(loginv1alpha1.RegisterDefaults(scheme))
 
+	// on incoming requests, restore the authenticator API group to the standard group
+	scheme.AddTypeDefaultingFunc(&loginv1alpha1.TokenCredentialRequest{}, func(obj interface{}) {
+		credentialRequest := obj.(*loginv1alpha1.TokenCredentialRequest)
+		if credentialRequest.Spec.Authenticator.APIGroup == nil {
+			return
+		}
+		restoredGroup, ok := groupsuffix.Unreplace(*credentialRequest.Spec.Authenticator.APIGroup, apiGroupSuffix)
+		if !ok {
+			return
```

Comment:
Is it worth putting a log message before these early `return`'s to help us debug in the future? I.e., if a login is failing with `"no such authenticator`", it might help to check and see that this defaulting func did what we wanted.

Reply:
Sure I can add one.